static/js/app.ts

```diff
@@ -1715,7 +1722,7 @@ function updatePageElements() {
     if (isSubmitted) {
       hand_in_btn?.setAttribute('disabled', 'disabled');
     } else {
-      hand_in_btn?.removeAttribute('disabled');
+      // hand_in_btn?.removeAttribute('disabled');
```

Comment:
The whole `else` statement should be removed, right?

Reply:
Nooo quite the opposite! Thanks for the catch!

Reply:
You are right, it is needed.